doorway-ui-components/src/text/Heading.tsx

```diff
@@ -5,6 +5,7 @@ export interface HeadingProps {
   className?: string
   priority?: number
   styleType?: HeaderType
+  id?: string
```

Comment:
Curious if there are any concerns with bringing this change into UI-C to allow for this functionality?